server/opendp_apps/dataverses/serializers.py

```diff
@@ -34,7 +34,7 @@ def save(self, **kwargs):
         return super().save()
 
 
-class DataverseHandoffSerializer(serializers.HyperlinkedModelSerializer):
+class DataverseHandoffSerializer(serializers.ModelSerializer):
```

Comment:
Line 27: Needs only two fields. `fields = [ 'user', 'dv_handoff']`
 - `dv_installation` is in the `dv_handoff` object

Reply:
`dv_installation` is not present in the `dv_handoff` object, though I am making this change now